examples/with-particles/README.md

```diff
@@ -0,0 +1,34 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
Let's use the same README template as other examples do, for example:

https://github.com/vercel/next.js/blob/canary/examples/with-eslint/README.md

Reply:
Fixed, thanks